### PR TITLE
Add 404 pages and fix styling on the 404 html template

### PR DESCRIPTION
**Pull request by SeverS**
## Description
- fixes styling for the 404 html template

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

**Review by quininez — DISMISSED** [review]

**Review by quininez — DISMISSED** [review]

File: extensions/apps/antenna/src/components/app-routes/not-found-component.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import ErrorLoader from '@akashaorg/design-system-core/lib/components/ErrorLoader';
+import { useTranslation } from 'react-i18next';
+import { useRootComponentProps } from '@akashaorg/ui-awf-hooks';
+import Button from '@akashaorg/design-system-core/lib/components/Button';
+
+export const NotFoundComponent = (props: { error?: Error; reset?: (key: string) => void }) => {
+  const { t } = useTranslation();
+  const { getRoutingPlugin, logger, worldConfig } = useRootComponentProps();
```
Comment (SeverS) [inline extensions/apps/antenna/src/components/app-routes/not-found-component.tsx]:
```suggestion
  const { getCorePlugins, logger, worldConfig } = useRootComponentProps();
```

File: extensions/apps/antenna/src/components/app-routes/not-found-component.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  useEffect(() => {
+    if (props.error) {
+      logger.warn(`Error in Antenna app: ${props.error}`);
+    }
+  }, [logger, props.error]);
+
+  const navigateHome = () => {
+    getRoutingPlugin().navigateTo({
```
Comment (SeverS) [inline extensions/apps/antenna/src/components/app-routes/not-found-component.tsx]:
```suggestion
    getCorePlugins().routing.navigateTo({
```

File: extensions/apps/auth-app/src/components/app-routes/not-found-component.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import ErrorLoader from '@akashaorg/design-system-core/lib/components/ErrorLoader';
+import { useTranslation } from 'react-i18next';
+import { useRootComponentProps } from '@akashaorg/ui-awf-hooks';
+import Button from '@akashaorg/design-system-core/lib/components/Button';
+
+export const NotFoundComponent = (props: { error?: Error; reset?: (key: string) => void }) => {
+  const { t } = useTranslation();
+  const { getRoutingPlugin, logger, worldConfig } = useRootComponentProps();
```
Comment (SeverS) [inline extensions/apps/auth-app/src/components/app-routes/not-found-component.tsx]:
```suggestion
  const { getCorePlugins, logger, worldConfig } = useRootComponentProps();
```

File: extensions/apps/auth-app/src/components/app-routes/not-found-component.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+  useEffect(() => {
+    if (props.error) {
+      logger.warn(`Error in Auth app: ${props.error}`);
+    }
+  }, [logger, props.error]);
+
+  const navigateHome = () => {
+    getRoutingPlugin().navigateTo({
```
Comment (SeverS) [inline extensions/apps/auth-app/src/components/app-routes/not-found-component.tsx]:
```suggestion
    getCorePlugins().routing.navigateTo({
```

File: extensions/apps/extensions/src/components/app-routes/not-found-component.tsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import ErrorLoader from '@akashaorg/design-system-core/lib/components/ErrorLoader';
+import { useTranslation } from 'react-i18next';
+import { useRootComponentProps } from '@akashaorg/ui-awf-hooks';
+import Button from '@akashaorg/design-system-core/lib/components/Button';
+
+export const NotFoundComponent = (props: { error?: Error; reset?: (key: string) => void }) => {
+  const { t } = useTranslation();
+  const { getRoutingPlugin, logger, worldConfig } = useRootComponentProps();
```
Comment (SeverS) [inline extensions/apps/extensions/src/components/app-routes/not-found-component.tsx]:
```suggestion
  const { getCorePlugins, logger, worldConfig } = useRootComponentProps();
```

File: extensions/apps/extensions/src/components/app-routes/not-found-component.tsx [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+  useEffect(() => {
+    if (props.error) {
+      logger.warn(`Error in Extensions app: ${props.error}`);
+    }
+  }, [logger, props.error]);
+
+  const navigateHome = () => {
+    getRoutingPlugin().navigateTo({
```
Comment (SeverS) [inline extensions/apps/extensions/src/components/app-routes/not-found-component.tsx]:
```suggestion
        getCorePlugins().routing.navigateTo({
```

File: extensions/apps/legal/src/components/app-routes/not-found-component.tsx [hunk 7]
```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import ErrorLoader from '@akashaorg/design-system-core/lib/components/ErrorLoader';
+import { useTranslation } from 'react-i18next';
+import { useRootComponentProps } from '@akashaorg/ui-awf-hooks';
+import Button from '@akashaorg/design-system-core/lib/components/Button';
+
+export const NotFoundComponent = (props: { error?: Error; reset?: (key: string) => void }) => {
+  const { t } = useTranslation();
+  const { getRoutingPlugin, logger, worldConfig } = useRootComponentProps();
```
Comment (SeverS) [inline extensions/apps/legal/src/components/app-routes/not-found-component.tsx]:
```suggestion
  const { getCorePlugins, logger, worldConfig } = useRootComponentProps();
```

File: extensions/apps/legal/src/components/app-routes/not-found-component.tsx [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+  useEffect(() => {
+    if (props.error) {
+      logger.warn(`Error in Legal app: ${props.error}`);
+    }
+  }, [logger, props.error]);
+
+  const navigateHome = () => {
+    getRoutingPlugin().navigateTo({
```
Comment (SeverS) [inline extensions/apps/legal/src/components/app-routes/not-found-component.tsx]:
```suggestion
    getCorePlugins().routing.navigateTo({
```

File: extensions/apps/notifications/src/components/app-routes/not-found-component.tsx [hunk 9]
```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import ErrorLoader from '@akashaorg/design-system-core/lib/components/ErrorLoader';
+import { useTranslation } from 'react-i18next';
+import { useRootComponentProps } from '@akashaorg/ui-awf-hooks';
+import Button from '@akashaorg/design-system-core/lib/components/Button';
+
+export const NotFoundComponent = (props: { error?: Error; reset?: (key: string) => void }) => {
+  const { t } = useTranslation();
+  const { getRoutingPlugin, logger, worldConfig } = useRootComponentProps();
```
Comment (SeverS) [inline extensions/apps/notifications/src/components/app-routes/not-found-component.tsx]:
```suggestion
  const { getCorePlugins, logger, worldConfig } = useRootComponentProps();
```

File: extensions/apps/notifications/src/components/app-routes/not-found-component.tsx [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+
+  useEffect(() => {
+    if (props.error) {
+      logger.warn(`Error in Notifications app: ${props.error}`);
+    }
+  }, [logger, props.error]);
+
+  const navigateHome = () => {
+    getRoutingPlugin().navigateTo({
```
Comment (SeverS) [inline extensions/apps/notifications/src/components/app-routes/not-found-component.tsx]:
```suggestion
    getCorePlugins().routing.navigateTo({
```

File: extensions/apps/profile/src/components/app-routes/not-found-component.tsx [hunk 11]
```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import ErrorLoader from '@akashaorg/design-system-core/lib/components/ErrorLoader';
+import { useTranslation } from 'react-i18next';
+import { useRootComponentProps } from '@akashaorg/ui-awf-hooks';
+import Button from '@akashaorg/design-system-core/lib/components/Button';
+
+export const NotFoundComponent = (props: { error?: Error; reset?: (key: string) => void }) => {
+  const { t } = useTranslation();
+  const { getRoutingPlugin, logger, worldConfig } = useRootComponentProps();
```
Comment (SeverS) [inline extensions/apps/profile/src/components/app-routes/not-found-component.tsx]:
```suggestion
  const { getCorePlugins, logger, worldConfig } = useRootComponentProps();
```

File: extensions/apps/profile/src/components/app-routes/not-found-component.tsx [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+
+  useEffect(() => {
+    if (props.error) {
+      logger.warn(`Error in Profile app: ${props.error}`);
+    }
+  }, [logger, props.error]);
+
+  const navigateHome = () => {
+    getRoutingPlugin().navigateTo({
```
Comment (SeverS) [inline extensions/apps/profile/src/components/app-routes/not-found-component.tsx]:
```suggestion
    getCorePlugins().routing.navigateTo({
```

File: extensions/apps/search/src/components/app-routes/not-found-component.tsx [hunk 13]
```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import ErrorLoader from '@akashaorg/design-system-core/lib/components/ErrorLoader';
+import { useTranslation } from 'react-i18next';
+import { useRootComponentProps } from '@akashaorg/ui-awf-hooks';
+import Button from '@akashaorg/design-system-core/lib/components/Button';
+
+export const NotFoundComponent = (props: { error?: Error; reset?: (key: string) => void }) => {
+  const { t } = useTranslation();
+  const { getRoutingPlugin, logger, worldConfig } = useRootComponentProps();
```
Comment (SeverS) [inline extensions/apps/search/src/components/app-routes/not-found-component.tsx]:
```suggestion
  const { getCorePlugins, logger, worldConfig } = useRootComponentProps();
```

File: extensions/apps/search/src/components/app-routes/not-found-component.tsx [hunk 14]
```diff
@@ -0,0 +1,36 @@
...
+
+  useEffect(() => {
+    if (props.error) {
+      logger.warn(`Error in Search app: ${props.error}`);
+    }
+  }, [logger, props.error]);
+
+  const navigateHome = () => {
+    getRoutingPlugin().navigateTo({
```
Comment (SeverS) [inline extensions/apps/search/src/components/app-routes/not-found-component.tsx]:
```suggestion
    getCorePlugins().routing.navigateTo({
```

File: extensions/apps/settings/src/components/app-routes/not-found-component.tsx [hunk 15]
```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import ErrorLoader from '@akashaorg/design-system-core/lib/components/ErrorLoader';
+import { useTranslation } from 'react-i18next';
+import { useRootComponentProps } from '@akashaorg/ui-awf-hooks';
+import Button from '@akashaorg/design-system-core/lib/components/Button';
+
+export const NotFoundComponent = (props: { error?: Error; reset?: (key: string) => void }) => {
+  const { t } = useTranslation();
+  const { getRoutingPlugin, logger, worldConfig } = useRootComponentProps();
```
Comment (SeverS) [inline extensions/apps/settings/src/components/app-routes/not-found-component.tsx]:
```suggestion
  const { getCorePlugins, logger, worldConfig } = useRootComponentProps();
```

File: extensions/apps/settings/src/components/app-routes/not-found-component.tsx [hunk 16]
```diff
@@ -0,0 +1,36 @@
...
+
+  useEffect(() => {
+    if (props.error) {
+      logger.warn(`Error in Settings app: ${props.error}`);
+    }
+  }, [logger, props.error]);
+
+  const navigateHome = () => {
+    getRoutingPlugin().navigateTo({
```
Comment (SeverS) [inline extensions/apps/settings/src/components/app-routes/not-found-component.tsx]:
```suggestion
    getCorePlugins().routing.navigateTo({
```

File: extensions/apps/vibes-console/src/components/app-routes/not-found-component.tsx [hunk 17]
```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import ErrorLoader from '@akashaorg/design-system-core/lib/components/ErrorLoader';
+import { useTranslation } from 'react-i18next';
+import { useRootComponentProps } from '@akashaorg/ui-awf-hooks';
+import Button from '@akashaorg/design-system-core/lib/components/Button';
+
+export const NotFoundComponent = (props: { error?: Error; reset?: (key: string) => void }) => {
+  const { t } = useTranslation();
+  const { getRoutingPlugin, logger, worldConfig } = useRootComponentProps();
```
Comment (SeverS) [inline extensions/apps/vibes-console/src/components/app-routes/not-found-component.tsx]:
```suggestion
  const { getCorePlugins, logger, worldConfig } = useRootComponentProps();
```

File: extensions/apps/vibes-console/src/components/app-routes/not-found-component.tsx [hunk 18]
```diff
@@ -0,0 +1,36 @@
...
+
+  useEffect(() => {
+    if (props.error) {
+      logger.warn(`Error in VibesConsole app: ${props.error}`);
+    }
+  }, [logger, props.error]);
+
+  const navigateHome = () => {
+    getRoutingPlugin().navigateTo({
```
Comment (SeverS) [inline extensions/apps/vibes-console/src/components/app-routes/not-found-component.tsx]:
```suggestion
    getCorePlugins().routing.navigateTo({
```

File: extensions/apps/vibes/src/components/app-routes/not-found-component.tsx [hunk 19]
```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import ErrorLoader from '@akashaorg/design-system-core/lib/components/ErrorLoader';
+import { useTranslation } from 'react-i18next';
+import { useRootComponentProps } from '@akashaorg/ui-awf-hooks';
+import Button from '@akashaorg/design-system-core/lib/components/Button';
+
+export const NotFoundComponent = (props: { error?: Error; reset?: (key: string) => void }) => {
+  const { t } = useTranslation();
+  const { getRoutingPlugin, logger, worldConfig } = useRootComponentProps();
```
Comment (SeverS) [inline extensions/apps/vibes/src/components/app-routes/not-found-component.tsx]:
```suggestion
  const { getCorePlugins, logger, worldConfig } = useRootComponentProps();
```

File: extensions/apps/vibes/src/components/app-routes/not-found-component.tsx [hunk 20]
```diff
@@ -0,0 +1,36 @@
...
+
+  useEffect(() => {
+    if (props.error) {
+      logger.warn(`Error in Vibes app: ${props.error}`);
+    }
+  }, [logger, props.error]);
+
+  const navigateHome = () => {
+    getRoutingPlugin().navigateTo({
```
Comment (SeverS) [inline extensions/apps/vibes/src/components/app-routes/not-found-component.tsx]:
```suggestion
    getCorePlugins().routing.navigateTo({
```

**Review by quininez — APPROVED** [review]

File: libs/design-system-components/src/components/NotificationsCard/index.tsx [hunk 21]
```diff
@@ -39,15 +39,15 @@ const NotificationsCard: React.FC<NotificationsCardProps> = props => {
...
+    mentionedPostLabel = 'mentioned you in a post',
+    mentionedCommentLabel = 'mentioned you in a comment',
+    replyToPostLabel = 'replied to your post',
+    replyToReplyLabel = 'replied to your reply',
+    followingLabel = 'is now following you',
+    repostLabel = 'reposted your post',
+    moderatedPostLabel = 'moderated your post',
+    moderatedReplyLabel = 'moderated your reply',
+    moderatedAccountLabel = 'suspended your account',
     emptyTitle,
```
Comment (josenriagu) [inline libs/design-system-components/src/components/NotificationsCard/index.tsx:51]: having these values set here as default, will make them not get translated if missed. could be better if those label props are not optional?

File: libs/design-system-components/src/components/SearchBar/index.tsx [hunk 22]
```diff
@@ -31,7 +31,7 @@ const SearchBar: React.FC<SearchBarProps> = props => {
     inputValue,
     customStyle,
     onInputChange,
-    inputPlaceholderLabel,
+    inputPlaceholderLabel = 'Search',
```
Comment (josenriagu) [inline libs/design-system-components/src/components/SearchBar/index.tsx:34]: same here

File: libs/design-system-components/src/components/TagProfileCard/index.tsx [hunk 23]
```diff
@@ -37,10 +37,10 @@ const TagProfileCard: React.FC<TagProfileCard> = props => {
...
     handleUnsubscribeTag,
-    subscribeLabel,
-    subscribedLabel,
-    unsubscribeLabel,
-    mentionsLabel,
+    mentionsLabel = 'Beams',
+    subscribeLabel = 'Subscribe',
+    subscribedLabel = 'Subscribed',
+    unsubscribeLabel = 'Unsubscribe',
     customStyle = '',
```
Comment (josenriagu) [inline libs/design-system-components/src/components/TagProfileCard/index.tsx:44]: here too

**Review by josenriagu — APPROVED** [review]

**Pull request merged**